etc/papd/print_cups.c

```diff
@@ -148,8 +167,6 @@ cups_printername_ok(char *name)         /* I - Name of printer */
 
         language = cupsLangDefault();
 
-	language = cupsLangDefault();
-
```

Comment:
Thanks for fixing this! It seems I messed up with a merge conflict resolution last year.